src/lexer.rs

```diff
@@ -589,10 +611,11 @@ fn atmosphere() -> Result<()> {
 #[test]
 fn comment() -> Result<()> {
     assert_eq!(
-        tokenize("abcd;+-12\t\n\r 12")?,
+        tokenize("abcd;+-12\n 12;dew\r34")?,
```

Comment:
removed \t ?

Reply:
Oh... this should be another commit

But this is because \t should not be a special character for comments, it does not break comments.